src/msg/async/AsyncConnection.h

```diff
@@ -71,7 +71,7 @@ class AsyncConnection : public Connection {
   void discard_out_queue();
   void discard_requeued_up_to(uint64_t seq);
   void requeue_sent();
-  int randomize_out_seq();
+  void randomize_out_seq();
```

Comment:
*Good*. I hate when people leave return values on functions that can only return success.